bricks/template/brick.yaml

```diff
@@ -21,8 +21,28 @@ environment:
 #  * an optional prompt phrase used when asking for the variable
 #  * a list of values (enums only)
 vars:
-  name:
+  package_name:
     type: string
-    description: Your name
-    default: Dash
-    prompt: What is your name?
+    description: Your package name
+    default: co.nimblehq.template
+    prompt: Package name?
+  app_name:
+    type: string
+    description: Your app name
+    default: Your App Name
+    prompt: App name?
+  project_name:
+    type: string
+    description: Your project name
+    default: your_project_name
+    prompt: Project name?
+  app_version:
+    type: string
+    description: Your app version
+    default: "0.1.0"
+    prompt: App version?
```

Comment:
@manh-t I think the prompt for app version and build number is odd since we can set them by default from scratch 🤔

Reply:
@doannimble it's a part of migration because we have the options to change app version and build number in the script
<img width="604" alt="image" src="https://user-images.githubusercontent.com/60863885/229064951-b521c167-4e8b-4ab0-99a5-448e40813a83.png">